### PR TITLE
Big Bunch of Typos

### PR DESCRIPTION
**Pull request by sadenar**
#### Summary
Content "Various typo, bad grammar and consistency fixes"

#### Purpose of change
I find typos when I translate stuff, typos annoy me, so I make a big list, check it twice and then fix a lot of typos in one go, this is one of those PRs.

#### Describe the solution
I fixed a few punctuation typos, reworked various descriptions that sounded weird to me, fixed weird automatically pluralized items, added more exclamation marks at the end of attacks, and applied all of the suggestions Brambor left in #68158.

#### Describe alternatives you've considered
- Not fixing stuff.
- Having somebody fix stuff for me.

#### Testing
Simple string changes shouldn't dramatically break the game, but I still ran my game once with them to test if any errors popped up, none did.

### CODE REVIEW

File: data/json/techniques.json [hunk 1]
```diff
@@ -1938,34 +1938,6 @@
     "defensive": true,
     "miss_recovery": true
   },
-  {
-    "type": "technique",
-    "id": "tec_muay_thai_rapid",
```
Comment (Maleclypse) [inline data/json/techniques.json:1943]: So the issue here is that you've deleted a bunch of muay thai stuff that was added before the pr was reopened.  And that is breaking tests.

Comment (GuardianDll) [reply data/json/techniques.json:1943]: @sadenar your best bet is copy the current version of `techniques.json`, paste it, and tweak the descriptions again

File: data/json/martialarts_fictional.json [hunk 2]
```diff
@@ -515,7 +515,7 @@
     "type": "technique",
     "id": "tec_venom_snake_bite",
     "name": "Viper Bite",
-    "messages": [ "You lash out at %s with a Viper Bite", "<npcname> lash out at %s with a Viper Bite" ],
+    "messages": [ "You lash out at %s with a Viper Bite!", "<npcname> lash out at %s with a Viper Bite!" ],
```
Comment (NetSysFire) [inline data/json/martialarts_fictional.json:518]:
```suggestion
    "messages": [ "You lash out at %s with a Viper Bite!", "<npcname> lashes out at %s with a Viper Bite!" ],
```

**Review by NetSysFire — APPROVED** [review]
Thanks for putting in the effort!

**Pull request merged**